### PR TITLE
Dataset update

### PR DESCRIPTION
**Pull request by Eugene-hu**
- Updates the validator dataset to `openwebtext`
- Adds additional forward command that augments the base prompt depending on the forward run
- Updates to the test

### CODE REVIEW

**Review by opentaco — APPROVED** [review]

File: openvalidators/forward.py [hunk 1]
```diff
@@ -384,6 +406,11 @@ async def forward(self):
         "answer_completions": answer_completions,
         "answer_times": [ans.elapsed_time for ans in answer_responses],
         "answer_rewards": answer_rewards.tolist(),
+        "augment_prompt": augment_prompt,
+        "augment_uids":augment_uids.tolist(),
+        "augment_completions":augment_completions,
```
Comment (steffencruz) [inline openvalidators/forward.py:411]: Please also include timing information
```python
"augment_times": [aug.elapsed_time for aug in augment_responses],
```

File: openvalidators/forward.py [hunk 2]
```diff
@@ -282,11 +284,30 @@ async def forward(self):
     start_time = time.time()
 
     # Load a base prompt from the dataset.
-    bootstrap_prompt = next(self.dataset)["context"]
+    data = next(self.dataset)["text"]
```
Comment (steffencruz) [inline openvalidators/forward.py:287]: So we grab the first 20 sentences?
Maybe more readable as 
```python
bootstrap_prompt = '.'.join(data.split('.',maxsplit=20)[:-1])
```

**Review by steffencruz — APPROVED** [review]

File: openvalidators/forward.py (hunk 1 above)
Comment (steffencruz) [reply openvalidators/forward.py:411]: Also followup_prompt is not stored.

**Comment by steffencruz** [comment]
@Eugene-hu can we close this?

**Pull request closed without merging**